modules/c++/nitf/source/TRE.cpp

```diff
@@ -112,12 +115,24 @@ nitf::Field TRE::getField(const std::string& key)
 {
     nitf_Field* field = nitf_TRE_getField(getNativeOrThrow(), key.c_str());
     if (!field)
-        throw except::NoSuchKeyException(Ctxt(FmtX(
-                "Field does not exist in TRE: %s", key.c_str())));
+        throw except::NoSuchKeyException(
+                Ctxt(FmtX("Field does not exist in TRE: %s", key.c_str())));
     return nitf::Field(field);
 }
 
-nitf::Field TRE::operator[] (const std::string& key)
+void TRE::updateFields()
+{
+    nitf_Error error;
+    if (!nitf_TREUtils_fillData(
+                getNative(),
+                ((nitf_TREPrivateData*)getNative()->priv)->description,
+                &error))
```

Comment:
Is there a reason for masking the member `error` variable in this scope?

Reply:
If there is, just make a comment as to why you're not following the pattern used elsewhere in the class